src/Cli/Microsoft.TemplateEngine.Cli/Commands/NewCommand.cs

```diff
@@ -44,6 +44,8 @@ internal NewCommand(
             Options.Add(SharedOptions.ProjectPathOption);
         }
 
+        public string DocsLink { get; } = "https://aka.ms/dotnet-new";
```

Comment:
Assigning the link inside here feels kinda wrong but the other option would involve a small breaking change, passing the link from the `NewCommandParser` down to this class.

```csharp
NewCommandParse -> NewCommandFactory -> NewCommand
``` 
Let me know what you think of this.

cc: @baronfel

Reply:
Totally fine to do this IMO.

Reply:
Thanks. I will run a local test and mark the PR to review